extras/Build/CMake/JUCEHelperTargets.cmake

```diff
@@ -52,11 +52,11 @@ add_library(juce::juce_recommended_lto_flags ALIAS juce_recommended_lto_flags)
 if((CMAKE_CXX_COMPILER_ID STREQUAL "MSVC") OR (CMAKE_CXX_SIMULATE_ID STREQUAL "MSVC"))
     target_compile_options(juce_recommended_lto_flags INTERFACE
         $<$<CONFIG:Release>:$<IF:$<STREQUAL:"${CMAKE_CXX_COMPILER_ID}","MSVC">,-GL,-flto>>)
-    target_link_libraries(juce_recommended_lto_flags INTERFACE
+    target_link_options(juce_recommended_lto_flags INTERFACE
```

Comment:
`target_link_options` was introduced in CMake 3.13, but JUCE requires CMake 3.12, so this code cannot use `target_link_options` yet.